code/game/objects/items/storage/book.dm

```diff
@@ -156,6 +156,27 @@ GLOBAL_LIST_INIT(bibleitemstates, list("bible", "koran", "scrapbook", "bible",
 			var/unholy2clean = A.reagents.get_reagent_amount("unholywater")
 			A.reagents.del_reagent("unholywater")
 			A.reagents.add_reagent("holywater",unholy2clean)
+		if(istype(A, /obj/item/twohanded/required/cult_bastard))
+			var/obj/item/twohanded/required/cult_bastard/sword = A
+			to_chat(user, "<span class='notice'>You begin to exorcise [sword].</span>")
+			playsound(src,'sound/hallucinations/veryfar_noise.ogg',40,1)
+			if(do_after(user, 40, target = sword))
+				playsound(src,'sound/effects/pray_chaplain.ogg',60,1)
+				for(var/obj/item/device/soulstone/SS in sword.shards)
+					for(var/mob/living/simple_animal/shade/EX in SS)
```

Comment:
this entire bit should just do this;
```
SS.usability = TRUE
SS.release_shades(user)
```

Reply:
That would turn them into neutral shards that anyone can use (and then have the purified 'use' them) but that wouldn't remove the shade's cult status right?